Automattic/blaze-ads
# Removes the custom translation code

### What and why? 🤔

We are removing the old translation code to start using the default process that DotOrg uses for all plugins.

Here are some references on how DotOrg works: 
* https://make.wordpress.org/meta/handbook/tutorials-guides/translations/
* https://make.wordpress.org/plugins/2015/12/06/plugin-translations-for-all-plugins/

I have included the current translated pot files in the plugin. According to the documentation, when the plugin is approved, those files will automatically be uploaded to translate.wordpress.org.

### Testing Steps ✍️

* Install the plugin and check that works fine
* Change the language of the WordPress admin page to another supported language. See the list in here: https://translate.wordpress.com/projects/woocommerce/blaze-ads
* Go to the plugins page `/wp-admin/plugins.php` and verify that you see the correct translation for our plugin
![Screenshot 2024-09-17 at 4 18 50 PM](https://github.com/user-attachments/assets/332e662f-b0d0-4f4a-8fd7-420830ac46b7)

### Review checklist
